UI/window-basic-main.cpp

```diff
@@ -2551,6 +2614,7 @@ void OBSBasic::ActivateAudioSource(OBSSource source)
 
 	bool vertical = config_get_bool(GetGlobalConfig(), "BasicWindow",
 			"VerticalVolControl");
+
```

Comment:
This whitespace change seems unrelated to the rest of the PR. You should revert this.